### ISSUE TITLE
Instagram handle dynamic item

### ISSUE THREAD

**Issue by marcandrews** [overlay-expert/help-desk#11]

**Comment by JackJallo** [comment]
I'd also consider adding YouTube handle and/or Discord handle as well.

**Comment by marcandrews** [comment]
@JackJallo We should make new issues for new requests.

**Comment by JackJallo** [comment]
Okay! Didn't wanna clog up topic space with similar ideas! Won't happen again. I think I did it, when I originally posted this one, with one or two others as well. My apologies!

**Comment by marcandrews** [comment]
No worries. The request is easier to track if it's on its own.

**Comment by JackJallo** [comment]
Definitely. I made a list of 5 others, including the one I commented here. But i'll stagger em out a bit so I'm not spamming the forum with requests. I appreciate your understanding!

**Comment by marcandrews** [comment]
Feel free to add them. If you don't, someone else will :sweat_smile:
